content/tutorials/marketplace-fees.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Fees
+description: The Ocean Protocol defines various fees for creating a sustainability loop.
+---
+
+## Path to sustainability
+
+Ocean Protocol is a community-driven project. Through OceanDAO, the project aims at achieving decentralized governance and sustainability loop. For the project to grow and improve, the participants are rewarded for their contributions. The Ocean Protocol's smart contracts include a fee mechanism to maintain the sustainability loop.
+
+## Fee types
+
+### Swap fee
+
+Swap fees are collected whenever someone swaps a datatoken for basetoken (e.g., OCEAN) or basetoken for datatoken. The swap could be inside a pool (using an automated market maker) or in a fixed-rate exchange.
+These are the fees that are applied whenever a user swaps basetoken or datatoken:
+
+- Publisher Marketplace swap fee
+- Consumer Marketplace swap fee
+- Provider Consumption Fees
+- [Ocean Community Fee](#ocean-community-fee)
+
+### Publish fee
+
+Publish fees will be charged to a publisher when they publish an asset.
+
+Currently, no fees are applicable in Ocean's default marketplace. However, it is possible to charge a publishing fee in the custom marketplace by adding an extra transaction in the publish flow.
+
+Based on the use case of the marketplace, the marketplace owner can decide if this fee should be charged or not.
+
+### Consume fee
+
+Consume fee is charged when a user holding datatoken exchanges it for a compute service, downloading a data asset or an algorithm.
+
+The following fees apply to the consumption of data assets as a part of Consume fee:
+
+- Consume Market Consumption Fee
+- Publisher Market Consumption Fee
+- Provider Consumption Fees
+- [Ocean Community Fee](#ocean-community-fee)
+
+### Ocean Community fee
+
+Ocean's smart contract collects the **Ocean Community fee** during a swap operation. These fees are reinvested in community projects via Ocean DAO and other initiatives. Currently, this fee is set to 0 (as long as either Ocean or H20 are used as the base token in the market) in the smart contract but can be updated by the Ocean Protocol foundation.
+
+Whereas a fee of 0.1% is applicable on all swaps using another token as the base token.
```

Comment:
OPC fees described in this paragraph are different from the amounts specified in table.

Here says `0%` for approved base tokens, `0.1%` for non-approved base tokens.
Table says `0.1%` for approved base tokens, `0.2%` for non-approved base tokens.

Reply:
Good catch @DMats. Changed the content now to reflect the table. o.1% and 0.2%